### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SerjDSA**
[DEMO LINK](https://serjdsa.github.io/react_people-table-advanced/)

### CODE REVIEW

File: src/utils/searchHelper.ts [hunk 1]
```diff
@@ -42,3 +41,11 @@ export function getSearchWith(
   // we return a string to use it inside links
   return newParams.toString();
 }
+
+export const createCenturiesParams = (centuries: string[], century: string) => {
+  return {
+    centuries: centuries.includes(century)
+      ? [...centuries.filter(c => c !== century)]
```
Comment (etojeDenys) [inline src/utils/searchHelper.ts:48]:
```suggestion
      ? centuries.filter(c => c !== century)
```
it's redundant, the filter method doesn't mutate the array

File: src/utils/peopleHelper.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        order ? b.name.localeCompare(a.name) : a.name.localeCompare(b.name),
+      );
+    case 'sex':
+      return people.sort((a, b) =>
+        order ? b.sex.localeCompare(a.sex) : a.sex.localeCompare(b.sex),
+      );
+    case 'born':
+      return people.sort((a, b) => (order ? b.born - a.born : a.born - b.born));
+    case 'died':
+      return people.sort((a, b) => (order ? b.died - a.died : a.died - b.died));
```
Comment (etojeDenys) [inline src/utils/peopleHelper.ts:20]:
```suggestion
    case 'born':
    case 'died':
      return people.sort((a, b) => (order ? b[sort]- a[sort] : a[sort] - b[sort]));
```
you can combine some cases

File: src/utils/peopleHelper.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  switch (sort) {
+    case 'name':
+      return people.sort((a, b) =>
+        order ? b.name.localeCompare(a.name) : a.name.localeCompare(b.name),
+      );
+    case 'sex':
+      return people.sort((a, b) =>
+        order ? b.sex.localeCompare(a.sex) : a.sex.localeCompare(b.sex),
+      );
```
Comment (etojeDenys) [inline src/utils/peopleHelper.ts:16]: same here

**Review by etojeDenys — CHANGES_REQUESTED** [review]
do not reset the filters on person click

https://github.com/mate-academy/react_people-table-advanced/assets/70220667/d6eba5c5-60c9-4894-a946-330df34abd19

**Review by SerjDSA — COMMENTED** [review]
[DEMO LINK](https://serjdsa.github.io/react_people-table-advanced/)

**Pull request closed without merging**